addons/sys_gui/XEH_postInit.sqf

```diff
@@ -41,9 +42,37 @@ DFUNC(enterVehicle) = {
     } else {
         // Close vehicle info display
         (QGVAR(vehicleInfo) call BIS_fnc_rscLayer) cutText ["", "PLAIN"];
+        //(QGVAR(antennaElevationInfo) call BIS_fnc_rscLayer) cutRsc [QGVAR(antennaElevationInfo), "PLAIN", 0, false];
     };
 };
 
+DFUNC(antennaElevationDisplay) = {
+    if (EGVAR(sys_core,automaticAntennaDirection)) exitWith {}; //TODO: Remove Display
+    
+    // Collect data from stance and antenna direction
+    private _stance = tolower (stance acre_player);
+    if (_stance == "" || _stance == "undefined") exitWith {};
+    private _antennaDirection = "_straight";
+    if (acre_player getVariable [QEGVAR(sys_core,antennaDirUp), false]) then {
+        _antennaDirection = "_bend";
+    };
+
+    // Check old antenna stance against new one
+    private _antennaStance = _stance + _antennaDirection;
+    if (GVAR(stanceCache) == _antennaStance) exitWith {};
+    GVAR(stanceCache) = _antennaStance;
+
+    // Change antenna icon to stance
+    private _ctrlGroup = uiNamespace getVariable ["ACRE_AntennaElevationInfo", controlNull];
+    if (isNull _ctrlGroup) exitWith {};
+
+    private _ctrl = _ctrlGroup controlsGroupCtrl 201;
+    _ctrl ctrlSetText "\idi\acre\addons\sys_gui\data\ui\" + _antennaStance + ".paa";
```

Comment:
Use `joinString`? Or whichever of `+`, `format`, `joinString` is faster. This function can run pretty often.

Reply:
`+` is acutally faster :)